Source/ending.py

```diff
@@ -0,0 +1,96 @@
+# Author: Siena Okuno
+# Project: Interactive Programming
+# Date: 10.30.2017
+# Description: Determines endings
+
+import pygame
+import sys
+
+def screenScroll(screen, color):
```

Comment:
The [Python convention](https://www.python.org/dev/peps/pep-0008/#function-names) is `screen_scroll`.